website/public/main.css

```
@@ -784,6 +784,14 @@ kbd.keystroke {
   }
 }
 
+/* Scrollable container */
+@media screen and (max-width: 670px) {
+  pre, code {
```

`overflow-x` and `max-height` are not appropriate for `code`. Only apply these rules to `pre`:
```suggestion
  pre {
```